docs/reference/setup-tools/kubeadm/kubeadm-init.md

```diff
@@ -40,6 +46,11 @@ following steps:
 
    Once control plane Pods are up and running, the `kubeadm init` sequence can continue.
 
+1. If kubeadm is invoked with `--feature-gates=DynamicKubeletConfig` enabled,
```

Comment:
... This functionality is now by default disabled as it is behind a feature gate, but is expected to be a default in future versions.